pbrEngine/engine.py

```diff
@@ -1331,6 +1331,8 @@ def _quitMatch(self):
         logger.debug("Quitting match")
         self._resetBlur()
         self.setVolume(0)  # Mute match setup beeping
+        self._matchFieldEffectStrength = 1.0
+        self._setFieldEffectStrength(self._matchFieldEffectStrength)
```

Comment:
what does it do?

Reply:
if the field effect strength is set to 0 the match prior, the next match will not have HP bars, by setting the field effect strength back to 1 when the match ends, it should fix that issue

Reply:
I feel like things like this should be done _before_ each match, not after

Reply:
it is done before each match at the current moment, which causes the HP bar issue

Reply:
to clarify, the field effect strength is set in _matchStartDelayed, the fov is set in a gevent spawned in _matchStart